libraries/messages/src/lib.rs

```diff
@@ -17,6 +17,9 @@ use ts_rs::TS;
 
 pub mod sender;
 pub mod sensor;
+
+pub const MAX_SIZE: usize = 85;
```

Comment:
Why 85?

Reply:
This is currently the max buffer size that is obtained when serializing the sbg struct.